### DISCUSSION TITLE
Suporte aos navegadores Mobile

### DISCUSSION THREAD

**Discussion by neysimoes** [frontendbr/forum#82] [Perguntas e Respostas]
No a partir do Android 4.4 o navegador nativo é o chrome e a do 4.3 para baixo o navegador padrão é um que nem nome tem.

Qual o android mais antigo que vocês dão suporte?

**Comment by felipefialho** [comment]
@neysimoes tomei a liberdade de alterar o titulo para não ficar tão extenso =). Tranquilo?

**Comment by neysimoes** [comment]
Tranquilo, coloquei a pergunta no corpo da issue.

**Comment by vitor-mendrone-deel** [comment]
Além da conversa que tivemos uma outra sugestão é olhar a tabela do caniuse.com

http://caniuse.com/usage-table

Somando todas as versões mais antigas que ainda tem usuários o total é de 1,48% 

2.3 - 0.02%
4 - 0.19%
4.1 - 0.39%
4.3 - 0,88%

Vale comparar com os dados gerados pelo Analytics do próprio site para o qual está realizando a pesquisa e ver se vale a pena investir tempo e grana no suporte.

**Comment by felipefialho** [comment]
Essa questão de o "quanto vale a pena investir tempo e grana no suporte" é fundamental.

Para uma empresa, sei lá, do porte da Americanas.com, 1% pode significar milhões de reais perdidos.

Já para uma empresa menor, pode não fazer diferença financeira alguma e dar prejuízo por causa da dificuldade no suporte.
